templates/web/base/waste/services.html

```diff
@@ -28,6 +28,9 @@
     [% IF is_staff AND NOT unit.orange_bag %]
         <a href="[% c.uri_for_action('waste/report', [ property.id ]) %]?additional=1&amp;service-[% unit.service_id %]=1" class="waste-service-link waste-service-descriptor">Request an additional [% unit.service_name FILTER lower %] collection</a>
     [% END %]
+    [% IF c.cobrand.waste_report_problem(unit.service_id) %]
```

Comment:
I think perhaps for now we just do NOT unit.orange_bag, no need for special sub-check. It's unclear from 'recycling' in their original message if that includes paper and card, but I don't see why it wouldn't, they could go missing also, might as well have everything to begin with.